static/scss/_variables.scss

```diff
@@ -7,16 +7,94 @@ $font-gray-light: #9c9c9c;
 $font-gray-disabled: rgba(0,0,0,.2);
 $link-text: #0074e1;
 
-//background colors
-$mm-brand-bg: #30bb5c;  // alt bb3097 058aca #2188e3 b13843 16b379 ca4444 a41e34 a23939
+/*
+// green 1
+
+$mm-brand-bg: #30bb5c;  // alt 3d68a9 30bb5c 42669c 058aca bb3097 058aca #2188e3 b13843 16b379 ca4444 a41e34 a23939
 $mm-brand-bg-dark: #30bb5c;  // alt #3884c7 b5424c 36af82 b53f53
 $mm-brand-bg-light: #30bb5c;
-$page-background: #eaeaea; 
+$brand-font: #30bb5c;
+$brand-overlay: rgba(28, 193, 81, 0.6);
+$page-background: #f1f1f1;
+$bg-light-green: #ccf0e5;
+$bg-light-blue: #cbe6f3;
+$bg-light-gray: #fbfbfb;
+$bg-med-gray: #ececec;
+$black-transparent: rgba(0, 0, 0, 0.9);
+
+
+// green 2
+$mm-brand-bg: #20b167;
+$mm-brand-bg-dark: #20b167;
+$mm-brand-bg-light: #3d68a9;
+$brand-font: #08b940;
+$brand-overlay: rgba(4, 173, 86, 0.6);
+$page-background: #f1f1f1;
+$bg-light-green: #ccf0e5;
+$bg-light-blue: #cbe6f3;
+$bg-light-gray: #fbfbfb;
+$bg-med-gray: #ececec;
+$black-transparent: rgba(0, 0, 0, 0.9);
+
+/*
+// salmon
+$mm-brand-bg: #ec6355;
+$mm-brand-bg-dark: #ec6355;
+$mm-brand-bg-light: #ec6355;
+$brand-font: #ec6355;
+$brand-overlay: rgba(236,99,85,.8);
+$page-background: #f1f1f1;
+$bg-light-green: #ccf0e5;
+$bg-light-blue: #cbe6f3;
+$bg-light-gray: #fbfbfb;
+$bg-med-gray: #ececec;
+$black-transparent: rgba(0, 0, 0, 0.9);
+
+// blue
+$mm-brand-bg: #3c66a5;
+$mm-brand-bg-dark: #3c66a5;
+$mm-brand-bg-light: #3c66a5;
+$brand-font: #3c66a5;
+$brand-overlay: rgba(38,104,204,0.5);
+$page-background: #f1f1f1;
+$bg-light-green: #ccf0e5;
+$bg-light-blue: #cbe6f3;
+$bg-light-gray: #fbfbfb;
+$bg-med-gray: #ececec;
+$black-transparent: rgba(0, 0, 0, 0.9);
+*/
```

Comment:
if we're ditching these colors I think we should just delete them (rather than commenting them out).